src/Cassandra/QueryProtocolOptions.cs

```diff
@@ -43,21 +43,11 @@ public enum QueryFlags
         public readonly int PageSize;
         public readonly ConsistencyLevel SerialConsistency;
 
-        private readonly long? _timestamp;
-
         public byte[] PagingState { get; set; }
         public object[] Values { get; private set; }
         public ConsistencyLevel Consistency { get; set; }
 
-        public DateTimeOffset? Timestamp
-        {
-            get
-            {
-                return _timestamp == null ? (DateTimeOffset?) null :
-                    TypeSerializer.UnixStart.AddTicks(_timestamp.Value * 10);
-            }
-        }
-
+        public long? Timestamp { get; }
```

Comment:
This is a breaking change. `QueryProtocolOptions` is a `public class` and this property is `public` so users might be using this property in their application code. 

The best way to go forward for now (without a major release) is to just make sure that this property is not used internally by the driver. I suggest adding an `internal` property called `RawTimestamp` or something similar that just returns the `_timestamp` field or we could make the new property an auto property and remove the field.